css/index.css

```diff
@@ -1,3 +1,537 @@
+        width: 400px;
+        margin: 0 auto;
+      }
+
+    .circle-img {
+        display: none;
+		}
+		
+	main, footer {
+		width: 700px;
+		}
+
+    .top-section{
+        margin-bottom: 15px;
+    }
+
+    .features {
+        width: 300px;
+    } 
+
+    .about {
+            width: 300px;
+            margin-left: 50px;
+        }
+
+	.middle-img{
+		width: 700px;
+    }
+    
+    .services p{
+        display: flex;
+        flex-wrap: wrap;
+        width: 200px;
+    }
+
+    .product p{
+        display: flex;
+        flex-wrap: wrap;
+        width: 200px;
+    }
+
+    .vision p{
+        display: flex;
+        flex-wrap: wrap;
+        width: 200px;
+    }
+
+	.bottom-section {
+		padding-bottom: 40px;
+        border-bottom: 1px solid black;
+        margin-bottom: 10px;
+	}
+    
+}
+
+/* At 500px (Phone) and below do the following: */
+@media screen and (max-width: 500px){
+	.container {
+        max-width: 500px;
+        margin: 0 auto;
+	}
+
+	header {
+		display: flex;
+		flex-direction: column-reverse;
+		align-items: center;
+        justify-content: center;
+        width: 500px;
+    }
+    
+	nav {
+	display: flex; 
+	flex-direction: column;
+	align-content: center;
+	justify-content: center;
+    width: 480px;}
+    
+    nav a {
+	display: flex;
+	text-align: center;
+	align-items: center;
+	justify-content: space-around;
+	padding: 20px 0;
+	color: rgb(188,188,188);
+	font-size: 20px;
+	width: 480px;
+	border-top: 1px solid rgb(188,188,188);
+	}
+	
+	#nav-services, #nav-product,  #nav-vision,  #nav-features, #nav-about, #nav-about, #nav-contact {
+	margin-left: -30px;
+	}
+
+	#nav-contact {
+		padding-bottom: 20px;
+		border-bottom: 1px solid rgb(188,188,188);
+	}
+	
+	img#logo {
+        margin-left: 0px;
+        padding-left: 0px;
+        margin-bottom: 20px;
+		padding-bottom: 10px;
+	}
+
+	.top-section {
+        width: 480px;
+        display: flex;
+    }
+
+    .features {
+        display: flex;
+        flex-wrap: wrap;
+        width: 500px;
+
+    }
+
+    .about {
+        display: flex;
+        flex-wrap: wrap;
+        width: 500px;
+        margin-left: 0px;
+    }
+
+
+	.middle-img {
+		width: 500px;
+	}
+
+    .top-section {
+        width: 500px;
+        display: flex;
+        flex-direction: column;
+    }
+	.bottom-section {
+        display: flex;
+        flex-direction: column;
+		width: 500px;
+		margin-top: 10px;
+		padding-bottom: 10px;
+		border-bottom: 1px solid black;margin-bottom: 10px;
+    }
+    
+    .services p{
+        display: flex;
+        flex-wrap: wrap;
+        width: 500px;
+    }
+
+    .product p{
+        display: flex;
+        flex-wrap: wrap;
+        width: 500px;
+    }
+
+    .vision p{
+        display: flex;
+        flex-wrap: wrap;
+        width: 500px;
+    }
+
+    .vision, .product {
+        margin-left: 0px;
+    }
+
+    main, footer {
+        width: 500px;
+        margin-left: 0px;
+        margin-right: 0px;
+    }
+
+}
```

Comment:
awesome job on the css. one thing I would work on is to try and have everything indented properly for cleaner code. If your using vscode there is a hotkey Alt + Shift + F and it will auto pretty it up